### DISCUSSION TITLE
Moto g23 (penangf) kernel source

### DISCUSSION THREAD

**Discussion by u8void** [community/community#123677] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Feature Area

Issues

### Body

Can you provide us the kernel source of Motorola g23 

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by ebndev** [comment]
Hi @ahmedshahin9,

Thanks for being a part of the GitHub Community, we're glad you're here!

I have moved this post to General, however, it looks like you're looking for support from a specific project. You might want to try asking for help somewhere that focuses on this project, such as opening an issue in the project's repository or seeing if they have a support page.

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place.

Best of luck!
